lib/reporting/identity_verification_report.rb

```diff
@@ -282,39 +323,45 @@ def data
           event = row['name']
           user_id = row['user_id']
           success = row['success']
+          gpo_verification_pending = row['gpo_verification_pending'] == '1'
+          in_person_verification_pending = row['in_person_verification_pending'] == '1'
+          fraud_review_pending = row['fraud_review_pending'] == '1'
 
-          event_users[event] << user_id
+          ignore_event_for_user =
+            fraud_review_pending &&
+            EVENTS_TO_IGNORE_IF_FRAUD_REVIEW_PENDING.include?(event)
+
+          users[event] << user_id unless ignore_event_for_user
+          users[sp_key(row)] << user_id if row['service_provider'].present?
```

Comment:
I renamed `user_events` to `users` here since holds sets of users across various dimensions, not just events. Here a specially formatted key is used to track users per-SP.